build/build-buildimages.sh

```diff
@@ -57,6 +57,11 @@ BuildAndTagStage python3.5-build
 BuildAndTagStage python3.6-build
 BuildAndTagStage python3.7-build
 BuildAndTagStage python
+BuildAndTagStage php-build-prereqs
+BuildAndTagStage php56-build
```

Comment:
php5.6-build, php7.0-build etc like we have for python

Reply:
This whole section is going to change after we decide what to do with PR #12